pygeoapi/util.py

```diff
@@ -282,6 +284,10 @@ def json_serial(obj):
             return base64.b64encode(obj)
     elif isinstance(obj, Decimal):
         return float(obj)
+    elif isinstance(obj, numpy.int64):
```

Comment:
To prevent importing numpy, suggest:

```python
    elif type(obj).__name__ == 'int64':
        return int(obj)
    elif type(obj).__name__ == 'float64':
        return float(obj)
```